app/forms/oregon_digital/fields/input_factory.rb

```diff
@@ -1,26 +1,18 @@
 module OregonDigital
   module Fields
     class InputFactory
-      class << self
-        def create(object, property)
-          decorate do
-            base_factory.create(object, property)
-          end
-        end
-
-        private
+      pattr_initialize :base_factory, :decorator
 
-        def decorate
-          decorator.new(yield)
+      def create(object, property)
+        decorate do
+          base_factory.create(object, property)
         end
+      end
 
-        def decorator
-          HasHintOption
-        end
+      private
 
-        def base_factory
-          HydraEditor::Fields::Factory
```

Comment:
@terrellt at the moment, it is THE core factory.  It could change, but at a minimum it should be defaulting to use that one unless overridden.

@straleyb Possibly, but its API is different, so I'm thinking at this point it's better to add decorations rather than subclass.